docs/standards/nft-2.0/LSP7-Digital-Asset.md

```diff
@@ -5,30 +5,93 @@ sidebar_position: 3
+
+To ensure a flexible and generic asset representation, the token contract should use the **[LSP4-DigitalAsset-Metadata](./LSP4-Digital-Asset-Metadata.md)**. In this way, any information could be attached to the token contract. 
+
+### Force Boolean
+
+It is expected in the LUKSO's ecosystem to use **smart contract based accounts** to operate on the blockchain, which includes receiving and sending tokens. EOAs can receive tokens but they will be mainly used to control these accounts and not to hold tokens.
+
+To ensure a **safe asset transfer**, an additional boolean parameter was added to the [transfer](../smart-contracts//lsp7-digital-asset.md#transfer) and mint functions where this parameter if set to **False**, the transfer will only pass if the recipient is a smart contract that implements the **[LSP1-UniversalReceiver](../generic-standards/lsp1-universal-receiver.md)** standard.
+
+![Token Force Boolean False](/img/standards/tokens-force-false.jpeg)
+
+:::note
+
+Implementing the **[LSP1-UniversalReceiver](../generic-standards/lsp1-universal-receiver.md)** standard will give a sign that the contract knows how to handle the tokens received.
+It's advised to set the **force** bool as **False** when transferring or minting tokens to avoid sending them to the wrong address.
+
+:::
+
+If set to **TRUE**, the transfer will not be dependent on the recipient, meaning **smart contracts** not implementing the **[LSP1-UniversalReceiver](../generic-standards/lsp1-universal-receiver.md)** standard and **EOAs** will be able to receive the tokens.
+
+![Token Force Boolean True](/img/standards/tokens-force-true.jpeg)
+
+
+### Token Hooks
+
+The current token standards do not allow **informing the sender** or **the recipient** about the transfer because the token contract acts like a **registry contract** that just tracks the balance of each address. This is causing a big problem where people will receive tokens and not know about it.
 
-LSP7 Digital Asset is an interface standard, meaning it creates a joint base to use and interact with such assets. Contracts and clients can query and transfer such assets in the same way, using the same methods.
+During an **ERC20 token transfer**, the balance of the sender is decreased and the balance of the recipient is increased without further interaction. 
 
-This standard was based on ERC20 and [ERC777](https://eips.ethereum.org/EIPS/eip-777).
+![ERC20 Transfer](/img/standards/erc20-transfer.jpeg)
 
-## Divisible _vs_ Non-Divisible
+On the opposite side, and during an **LSP7 token transfer**, the same mechanism is applied of increasing the balance of the recipient and decreasing the balance of the sender, with an additional action, which is **informing** the sender and the recipient by calling the   **[`universalReceiever(...)`](../generic-standards/lsp1-universal-receiver.md#lsp1---universal-receiver)** function on their profiles.
```

Comment:
During an **LSP7 token transfer**, as well as updating the balances, both the sender and recipient are informed of the transfer via the **[`universalReceiver(...)`](../generic-standards/lsp1-universal-receiver.md#lsp1---universal-receiver)** function on their profiles (being called by the LSP7 Token Contract).